src/store.rs

```diff
@@ -262,4 +267,33 @@ mod tests {
 
         assert_eq!(updated_ticket.description, draft.description);
     }
+
+    #[test]
+    fn updating_ticket_status_should_change_ticket_to_new_status() {
+        let faker = fake::Faker;
+
+        //arrange
+        let draft = TicketDraft {
+            title: Title::new(faker.fake()).expect("Failed to get a title"),
+            description: faker.fake(),
+        };
+
+        let mut ticket_store = TicketStore::new();
+
+        let ticket_id = ticket_store.create(draft.clone());
```

Comment:
You can use `generate_and_persist_ticket` to avoid duplicating code in every test case.

Reply:
Updated.